src/main/java/com/minecolonies/colony/ColonyView.java

```diff
@@ -289,6 +298,30 @@ public IMessage handlePermissionsViewMessage(ByteBuf buf)
     }
 
     /**
+     * Update a ColonyView's workOrders given a network data ColonyView update packet.
+     * This uses a full-replacement - workOrders do not get updated and are instead overwritten.
+     *
+     * @param buf       Network data
+     * @param order     is 0 if this is the first message after a change.
+     * @return          null == no response
+     */
+    public IMessage handleColonyViewWorkOrderMessage(ByteBuf buf, int order, ColonyViewWorkOrderMessage.workOrderMessages operation)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): operation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)